webapp/src/components/sidebar_right/github_items.jsx

```diff
@@ -7,10 +7,32 @@ import PropTypes from 'prop-types';
 import {Badge} from 'react-bootstrap';
 import {makeStyleFromTheme, changeOpacity} from 'mattermost-redux/utils/theme_utils';
 
+//retain previous item to show deleted items between 2 refresh call
+let previousItems = [];
+let previousStateRhs;
+
 function GithubItems(props) {
     const style = getStyle(props.theme);
-
-    return props.items.length > 0 ? props.items.map((item) => {
+    let oldElements = [];
+    let newElements = [];
+
+    if (previousStateRhs === props.rhsState) {
+        oldElements = previousItems.map((item) => {
+            if (!props.items.find((i) => i.id === item.id)) {
+                return {...item, missing: true};
+            }
+            return item;
+        });
+        newElements = props.items.filter((item) => !previousItems.find((i) => i.id === item.id));
+    } else {
+        previousStateRhs = props.rhsState;
+        newElements = [...props.items];
+    }
+
+    const elements = [...oldElements, ...newElements];
+    previousItems = [...elements];
+
+    return elements.length > 0 ? elements.map((item) => {
```

Comment:
Pre-existing code, but could we eliminate the ternary here and just return early?
```js
if (elements.length == 0) {
    return <div style={style.container}>{'You have no active items'}</div>;
}
```